integrations/deployment/flatpak.rst

```diff
@@ -19,4 +19,47 @@ The `packaging process <http://docs.flatpak.org/en/latest/first-build.html>`__ i
 - Run the ``flatpak-builder`` in order to produce the application
 - `publish <http://docs.flatpak.org/en/latest/publishing.html>`__ the application for further distribution
 
+With help of conan's :ref:`json generator<deployable_json_generator>`, the manifest creation could be easily automated. For example, the custom script could generate ``build-commands`` and ``sources`` entries within the manifest file:
```

Comment:
Please add a link to the manifest file format or to a reference page in the flatpak's docs

Reply:
added a link to added a link to http://docs.flatpak.org/en/latest/manifests.html